x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/IndicesAndAliasesResolver.java

```diff
@@ -129,6 +129,7 @@ ResolvedIndices resolveIndicesAndAliases(String action, IndicesRequest indicesRe
             final boolean replaceWildcards = indicesOptions.expandWildcardsOpen() || indicesOptions.expandWildcardsClosed();
 
             // check for all and return list of authorized indices
+//            if (isAllIndices(indicesRequest.indices())) {
             if (IndexNameExpressionResolver.isAllIndices(indicesList(indicesRequest.indices()))) {
```

Comment:
```suggestion
            if (isAllIndices(indicesRequest.indices())) {
```
I assume this is the proposed change.

Reply:
Yes you are right. 🤦‍♂️ I must have been testing and then committed the test version.